src/components/Header.tsx

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+interface HeaderProps {
+  newTitle: string;
+  onTitleChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  onAddTodo: (e: React.FormEvent<HTMLFormElement>) => void;
```

Comment:
Your typings are too close to the actual implementation, while they should reflect the logic

The "title" looks confusing here
You may use AI to pick variable names - just describe your current logic to the chat and ask it to suggest variable names (my CTO does so regularly)

```suggestion
  newTodoText: string;
  onNewTodoTextChange: (value: string) => void;
  onAddTodo: (newTodoText: string) => void;
```

Reply:
Thank you. Fixed